api/pkg/parser/catalog.go

```diff
@@ -418,6 +418,7 @@ func typeForKind(kind string) (tektonKind, error) {
 	switch kind {
 	case "Task":
 		return &v1beta1.Task{}, nil
+	// ClusterTask support is deprecated in Pipeline 0.47.0 and will be remove from Hub latter
```

Comment:
clustertasks are deprecated in 0.44

nit: `remove` -> `removed`

`latter` -> `in future releases`